core-bundle/contao/drivers/DC_Folder.php

```diff
@@ -303,12 +303,21 @@ public function showAll()
 				$this->isValid($this->intId);
 			}
 
+			$children = Input::get('children');
+
+			// Backwards compatibility
+			if (Input::get('childs') !== null)
+			{
+				trigger_deprecation('contao/core-bundle', '5.3', 'Using the "childs" query parameter has been deprecated and will no longer work in Contao 6. Use the "children" parameter instead.');
+				$children = Input::get('childs');
+			}
+
 			$arrClipboard = $objSession->get('CLIPBOARD');
 
 			$arrClipboard[$this->strTable] = array
 			(
 				'id' => $this->urlEncode($this->intId),
-				'childs' => Input::get('childs'),
+				'children' => $children,
```

Comment:
Can we rename this without breaking BC? 
AFAIK `arrClipboard` gets passed to some callbacks and `$objSession->get('CLIPBOARD')` is also used by some extensions I think.

How about keeping both and removing `childs` in Contao 6.0?

Reply:
Oh yes, good catch. 👍 Fixed in a3637ae1e049b0d0417426376f05d20e33505539.